pkgs/development/compilers/julia/patches/1.12/0001-skip-failing-and-flaky-tests.patch

```diff
@@ -0,0 +1,25 @@
+From e26b82d0c162b6c22b65b2a5d2cd03d98fd9f5d4 Mon Sep 17 00:00:00 2001
+From: Sergey Volkov <taranarmo@gmail.com>
+Date: Thu, 9 Oct 2025 13:00:31 +0200
+Subject: [PATCH] disable failing and flaky tests
+
+---
+ test/Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/test/Makefile b/test/Makefile
+index 69b7ad1451..583e64a287 100644
+--- a/test/Makefile
++++ b/test/Makefile
+@@ -30,7 +30,7 @@ default:
+ 
+ $(TESTS):
+ 	@cd $(SRCDIR) && \
+-	$(call PRINT_JULIA, $(call spawn,$(JULIA_EXECUTABLE)) --check-bounds=yes --startup-file=no --depwarn=error ./runtests.jl $@)
++	$(call PRINT_JULIA, $(call spawn,$(JULIA_EXECUTABLE)) --check-bounds=yes --startup-file=no --depwarn=error ./runtests.jl --skip NetworkOptions REPL channels FileWatching ccall loading cmdlineargs Distributed precompile compileall $@)
```

Comment:
I wonder if we should do something similar to `julia-bin`... (I'm not a maintainer of the source-built `julia` though)